keyboards/redscarf_iiplus/verd/info.json

```
@@ -0,0 +1,100 @@
+{
+  "keyboard_name": "Red Scarf II+ Ver D",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 19,
+  "height": 5,
+  "layouts": {  
+    "LAYOUT": {

```

```suggestion
    "LAYOUT_60_numpad_split0": {
```

And we also need another tree here for the non-split-0 layout :)